prow/plugins/label/label.go

```diff
@@ -109,13 +161,14 @@ func handle(gc githubClient, log *logrus.Entry, ic github.IssueCommentEvent) err
 		}
 
 	}
-
+	
 	if len(nonexistent) > 0 {
 		msg := fmt.Sprintf(nonExistentLabel, strings.Join(nonexistent, ", "))
-		if err := gc.CreateComment(owner, repo, number, plugins.FormatICResponse(ic.Comment, msg)); err != nil {
+		if err := gc.CreateComment(owner, repo, number, plugins.FormatICResponse(ae.comment, msg)); err != nil {
+		//if err := gc.CreateComment(owner, repo, number, plugins.FormatResponseRaw(ae.body, ae.url, ae.login, msg)); err != nil {	
```

Comment:
"FormatResponseRaw" may be better than "FormatICResponse" as we now cater for issue comment, issue, and pull request.